eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -314,6 +327,8 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.setBoolean(6, m.isMultiPart)
         statement.setString(7, m.experimentName)
         statement.setString(8, m.recipientNodeId.value.toHex)
+        statement.setString(9, m.paymentHash.toHex)
+        statement.setString(10, serialization.write(m.routingHints))
```

Comment:
Note to self: we are using `writePretty` in `PgChannelsDb`/`PgNetworkDb` which is useless since `jsonb` doesn't preserve formatting.

Reply:
Done in #2279.